src/test/scala/example/HelloSpec.scala

```diff
@@ -0,0 +1,44 @@
+package example
+
+import example.Animal.{Lion, Tiger}
+import example.Zoo.{GreetingZoo, NorthZoo}
+import org.scalatest.matchers.should.Matchers
+import org.scalatest.wordspec.AnyWordSpecLike
+
+class HelloSpec extends AnyWordSpecLike with Matchers {
+
+  "BorerAkkaSerializer" should {
+    "serialize singleton" in {
+      val obj = Tiger
+      val serializer = new BorerAkkaSerializer()
+      val serialized = serializer.toBinary(obj)
```

Comment:
Actually, can you use `SerializationTestKit` from Akka here? so that we see that Akka indeed dispatches the serialization to Borer (coz here Borer is called directly)

Reply:
Plus disable fallback to Java serialization (https://doc.akka.io/docs/akka/2.5.32//serialization.html#enable-additional-bindings) to make sure stuff is serialized with Borer, or an exception is thrown, no third option